test/test_integrity_test.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
+
+import pytest
+from AxonDeepSeg.integrity_test import integrity_test
+
+
+class TestCore(object):
+    def setup(self):
+        pass
+
+    def teardown(self):
+        pass
+
+    # --------------integrity_test.py tests-------------- #
+    @pytest.mark.integration
+    def test_integrity_test_script_runs_succesfully(self):
```

Comment:
typo: test_integrity_test_script_runs_successfully

Reply:
Haha woops – I'd say it was a typo but looks like I just forgot how to properly spell "succes**s**fully" 😅. Thanks